client/lib/plans/constants.js

```diff
@@ -118,3 +182,11 @@ export const getPlanObject = planName => {
 
 	return objectPlan;
 };
+
+export const getPlanFeaturesObject = planName => {
+	const planFeatures = filter( featuresList, obj =>
+		obj.plans.indexOf( planName ) !== -1
+	);
+
+	return sortBy( planFeatures, 'order' );
```

Comment:
I understand what you're doing here, but this may be somewhat difficult to maintain in the future. Maybe add a list of feature references to the plan object instead?

e.g.
```javascript
	[ PLAN_FREE ]: {
        getFeatures: () => [ FEATURE_3GB_STORAGE, FEATURE_WP_SUBDOMAIN,  ... ]
		getTitle: () => i18n.translate( 'Free' ),
		getPriceTitle: () => i18n.translate( 'Free for life' ),
		getProductId: () => 1,
		getDescription: () => i18n.translate( 'Get a free blog and be on your way to publishing your first post in less than five minutes.' )
	},
```